launch/launch/frontend/parser.py

```diff
@@ -138,6 +141,15 @@ def get_parser_from_extension(
         except KeyError:
             raise RuntimeError('Not recognized frontend implementation')
 
+    @classmethod
+    def get_file_extensions_from_parsers(cls) -> Set[Type['Parser']]:
+        """Return a set of file extensions gathered from the parser implementations."""
+        cls.load_parser_implementations()
+        return set(itertools.chain.from_iterable([
+            parser_extension.get_file_extensions()
+            for parser_extension in cls.frontend_parsers.values()
+        ]))
```

Comment:
@christophebedard nit: no need for the temporary list
```suggestion
        return set(itertools.chain.from_iterable(
            parser_extension.get_file_extensions()
            for parser_extension in cls.frontend_parsers.values()
        ))
```

Reply:
fixed in 7fae1d4e3ac33ab32b9626ddb53c85cb1c0f5191